### ISSUE TITLE
Vector sign shifts in LaTeX or markdown cells

### ISSUE THREAD

**Issue by ivanzolotov** [HappenApps/Quiver#926]
Some problem with 3.0.6 and 3.0.7 trial on OS X 10.12.6. When I try something like this: `$ \vec E $` in LaTeX or Markdown cell, the result is disappointing:
![Vector sign shifts](http://share.ivanzolotov.s3.amazonaws.com/2017/20170731172823.png)

**Comment by ivanzolotov** [comment]
It's 3.1.1, but the issue is still here. All my content is broken and looks awful:

> ![](http://share.ivanzolotov.s3.amazonaws.com/2017/20171106004650.png)

Are you going to fix the issue? Are there some way to get older version of Quiver, where this problem is not yet appeared?

**Comment by ylian** [comment]
@ivanzolotov Yes I will fix this. The change was due to updated WebKit, so older versions of Quiver won't work either.

**Comment by ivanzolotov** [comment]
Good news!

**Comment by Phy-David-Zhang** [comment]
A temporary solution: add the following lines in the application-wide macros under the LaTeX cell preferences

```
\renewcommand{\hat}[1]{\overset{\;\!\large\unicode{813}}{#1}}
\renewcommand{\vec}[1]{\overset{\;\!\large\unicode{8431}}{#1}}
```

and delete the lines after the update is available.

**Comment by ivanzolotov** [comment]
Works great! Thank you.

**Comment by ylian** [comment]
Fixed in 3.1.2.
